scss/_modal.scss

```diff
@@ -39,8 +52,16 @@
 // Shell div to position the modal with bottom padding
 .modal-dialog {
   position: relative;
-  width: auto;
+  display: inline-block;
+  width: calc(100% - 20px);
+
+  // Stops modal overflowing on mobiles
+  max-width: calc(100% - 20px);
+
   margin: 10px;
+  text-align: left;
```

Comment:
Should probably have a comment explaining that this `text-align` and `white-space` are here to undo the ones in `.modal::before`